test/external/imports/ts-cjs/tsconfig.json

```diff
@@ -4,12 +4,11 @@
 	"compilerOptions": {
 		"target": "esnext",
 		"module": "commonjs",
-		"declaration": true,
+		"declaration": false,
 		"sourceMap": false,
```

Comment:
Default is [false](https://www.typescriptlang.org/tsconfig#sourceMap), this line can be deleted

Reply:
Will do